### PR TITLE
Adding a gradio demo of SVD to be run locally

### PR DESCRIPTION
**Pull request by yvrjsharma**
added gradio_app.py, updated requirements/pt2.txt and readme.md for relevant libraries and information respectively

### CODE REVIEW

File: requirements/pt2.txt [hunk 1]
```diff
@@ -37,3 +37,6 @@ wandb>=0.15.6
 webdataset>=0.2.33
 wheel>=0.41.0
 xformers>=0.0.20
+fire
+uuid
```
Comment (akx) [inline requirements/pt2.txt:41]: uuid is a standard library, it doesn't need to be a requirement.

File: scripts/demo/gradio_app.py [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+import os
+from glob import glob
+from pathlib import Path
+from typing import Optional
+
+import cv2
+import numpy as np
+import torch
+from einops import rearrange, repeat
+from fire import Fire
```
Comment (akx) [inline scripts/demo/gradio_app.py:11]: This import (and the dependency) is unused.

**Comment by timudk** [comment]
Awesome thank you so much @yvrjsharma!! This is amazing and so fast!

Before merging I have a couple of requests:
- add gradio to reqs (and remove fire, uuid as @akx suggested)
- only download ckpt if not already present (it redownloaded it for me)
- it would be nice to add options for all models (instead of only svd-xt), it's optional but if not we should rename the file
- fps_id should be an int (but slider is float)
- a lot of code seems to be copy & pasted from `simple_video_sample`, can we instead just import functions we need?

**Comment by Sumith1896** [comment]
@yvrjsharma thanks for the PR! Could you please address @timudk's comments from above and let us know? One of us will have look (cc @AK391)

**Comment by yvrjsharma** [comment]
Hi, @timudk @Sumith1896, sincere apologies I couldn't come back to this thread any earlier due to various reasons. 
I have addressed all the comments from above. Thanks for the detailed comments and for following up on this! Let me know if this PR need anything else from my side.

**Review by Sumith1896 — APPROVED** [review]
tried locally and works, thanks for the PR @yvrjsharma!

**Pull request merged**